CONTRIBUTING.md

```diff
@@ -83,13 +101,50 @@ npm run dev:pristine
 npm run dev:format
 ```
 
+## Coding conventions
```

Comment:
Should we combine the "Coding rules" and "Coding conventions" sections? Seems kinda odd to break these into 2 sections. I think we should just delete the "Coding conventions" header and add the two items here as the 4th and 5th items in the list under Coding rules.

Reply:
Done